common_crawler/main.py

```diff
@@ -149,6 +222,14 @@ def process_crawl_and_upload(
     if not common_crawl_result.url_results:
         print("No url results found. Ceasing main execution.")
         return common_crawl_result
+
+    print("Removing urls already in the database")
```

Comment:
I do want to say I very much appreciate these status prints: They're clear, concise, and make it easier to see what's going on without having to plumb through the code.